app/config/config.yml

```diff
@@ -69,6 +69,9 @@ swiftmailer:
 
 # MassiveSearch Configuration
 massive_search:
+    persistence:
+        doctrine_orm:
+            enabled: true
```

Comment:
Shouldn't that be the default value? Then we wouldn't have to add it to the configuration...